src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/Text/Text.jsx

```diff
@@ -6,16 +6,27 @@ import {Input} from '../../../../../../DesignSystem/Input';
 
 export class Text extends React.Component {
     render() {
-        let {field, id} = this.props;
+        let {field, id, editorContext} = this.props;
         let {values, handleChange, handleBlur} = this.props.formik;
 
+        const fieldType = field.jcrDefinition.requiredType;
+        const isNumber = fieldType === 'DOUBLE' ||
+            fieldType === 'LONG' ||
+            fieldType === 'DECIMAL';
+
+        const decimalSeparator = editorContext.lang === 'en' ? ',' : '.';
```

Comment:
des tests sur ce genre de chose ?

Reply:
Yes, why not, I think it's more persistent from end-to-end, but no problem I'll add some tests.